backend/database/schema.sql

```diff
@@ -0,0 +1,110 @@
+create type role_type as enum ('admin', 'member');
+create type colour_type as enum ('red', 'orange', 'yellow', 'blue', 'green', 'grey', 'brown', 'purple', 'pink');
+create type checkout_type as enum ('borrow', 'use');
+create type checkout_status as enum ('waiting', 'checked_out', 'returned');
+create type approval_status as enum ('pending_approval', 'approved', 'not_approved');
+
+CREATE TABLE person (
+    zid             varchar(7) PRIMARY KEY,
```

Comment:
If this is used throughout, consider making a separate type called `zid` which is essentially made as another name for `varchar(7)`.

https://www.postgresql.org/docs/12/sql-createtype.html